lago/prefix.py

```diff
@@ -742,7 +742,7 @@ def _ova_to_spec(self, filename):
                           "template_type": "qcow2",
                           "format": "qcow2",
                           "dev": "vda",
-                          "name": "root",
+                          # "name": "root",
```

Comment:
Though I agree that the second makes more sense, not sure if we are using multi-image ovas anywhere though.